testing/snapshot-test.md

```diff
@@ -0,0 +1,20 @@
+# Snapshot test
+
+## Introduction
+Snapshot tests are used to test visual regression of presentational components and for reducers in order to avoid deep comparison boilerplate code
+
+## Dos
+- The visual component output has to be relatively small.  
+- Focus to shallow snapshot comparison.
+- In cases where snapshot output is not readable use enzyme-to-json utility.
```

Comment:
Don't we always need to use the utility? AFAIK, snapshot output is not readable without it (i might be wrong though)

Reply:
Not always, although don't know exact reasons why sometimes we need it

Reply:
always needed for react component (from my understanding)